google-cloud-spanner/src/main/java/com/google/cloud/spanner/InstanceInfo.java

```diff
@@ -76,6 +80,10 @@ static FieldMask toFieldMask(InstanceField... fields) {
 
     public abstract Builder putAllLabels(Map<String, String> labels);
 
+    public abstract Builder addEndpointUri(String endpointUrl);
```

Comment:
Same as above: Would it be possible to make these package-private? Or do they need to be public because it is used by client library code residing in other packages?

Reply:
See comment above.. this is to keep same style